examples/events.rs

```diff
@@ -15,13 +15,13 @@ use linemux::MuxedEvents;
 pub async fn main() {
     let args: Vec<String> = std::env::args().skip(1).collect();
 
-    let mut events = MuxedEvents::new();
+    let mut events = MuxedEvents::new().unwrap();
```

Comment:
I forget myself quite frequently, but as of Rust 2018 the recommended way to do examples is to return a `Result` from `main`. You can just do `Result<(), Box<dyn std::error::Error>>` as the return type to avoid having to do any extra conversions too, but the idea is to encourage people copy-pasting code to not leave `unwrap` all over the place.